Mathlib/NumberTheory/NumberField/FinitePlaces.lean

```diff
@@ -75,11 +77,13 @@ section FinitePlace
 variable {K : Type*} [Field K] [NumberField K] (v : HeightOneSpectrum (𝓞 K))
 
 /-- The embedding of a number field inside its completion with respect to `v`. -/
-def embedding : K →+* adicCompletion K v :=
-  @UniformSpace.Completion.coeRingHom K _ v.adicValued.toUniformSpace _ _
+noncomputable def embedding : WithVal (v.valuation K) →+* adicCompletion K v :=
```

Comment:
We're just in the namespace `NumberField`, so I think that the word "finite" should appear somewhere in this `embedding` (perhaps by putting everyting below in a `FinitePlace` namespace?)